app/assets/javascripts/num-players-slider.js

```diff
@@ -0,0 +1,73 @@
+let sliders = []
+
+document.addEventListener("turbolink:before-cache", function() {
+  if (sliders.length == 0) return;
+
+  sliders.forEach(slider => { slider.noUiSlider.destroy() });
+
+  sliders = []
+})
+
+document.addEventListener("turbolinks:load", function() {
+  const elements = document.querySelectorAll(".num-player-slider");
+  elements.forEach(function (element) {
+    if (!element) return;
+    let startMin = 1;
+    let startMax = 12;
+    if (element.dataset.minPlayers) {
+      startMin = element.dataset.minPlayers;
+    }
+    if (element.dataset.maxPlayers) {
+      startMax = element.dataset.maxPlayers;
+    }
+
+    noUiSlider.create(element, {
+      start: [startMin, startMax],
+      connect: true,
+      orientation: 'horizontal',
+      range: {
+        'min': 1,
+        'max': 12
+      },
+      step: 1,
+      pips: {
+        mode: 'steps'
+      },
+      behaviour: 'tap-drag'
+    });
+
+    switch (element.dataset.type) {
```

Comment:
I'm not a big fan of switch cases, especially when it's only 2 cases. 
A nicer solution might be:
```
if (element.dataset.type == "post") element.noUiSlider.on("set", postOnSliderUpdate)
if (element.dataset.type == "filter") element.noUiSlider.on("set", filterOnSliderUpdate)
```

Reply:
I made it a switch case so that if more cases needed to be added, I could do so easily. However, since it doesn't seem like more than two cases are needed, I agree the switch statement is overkill.